src/3d/qgspointcloudlayer3drenderer.cpp

```diff
@@ -73,7 +79,12 @@ Qt3DCore::QEntity *QgsPointCloudLayer3DRenderer::createEntity( const Qgs3DMapSet
   if ( !pcl || !pcl->dataProvider() || !pcl->dataProvider()->index() )
     return nullptr;
 
-  return new QgsPointCloudLayerChunkedEntity( pcl->dataProvider()->index(), map );
+  return new QgsPointCloudLayerChunkedEntity( pcl->dataProvider()->index(), map, mSymbol );
```

Comment:
This should clone the symbol, and have QgsPointCloudLayerChunkedEntity take ownership of the passed symbol